### PR TITLE
added links to builds

### PR DESCRIPTION
**Pull request by GoodBoyDigital**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +43,20 @@ Pixi.js has full [WebGL](https://en.wikipedia.org/wiki/WebGL) support and seamle
...
 
-### Installation ###
+### Setup ###
 
-Pixi.js can be installed with [Bower](https://bower.io/#getting-started), [NPM](https://docs.npmjs.com/getting-started/what-is-npm) or simply using a content delivery network (CDN) URL to embed Pixi.js directly on your HTML page. 
+Its easy to get started with Pixi.js! Simple grab the pre-built versions from here:
+
+Master - Nice and stable Pixi.js
+- Unminified: [http://pixijs.download/master/pixi.js]
+- Minified: [http://pixijs.download/master/pixi.min.js]
```
Comment (bigtimebuddy) [inline README.md]: I would change this to use the **release** branch instead **http://pixijs.download/release/pixi.js**, this will be the most stable.

Comment (GoodBoyDigital) [inline README.md]: good shout! tweaked 👍

**Comment by bigtimebuddy** [comment]
@englercj is working on getting the SSL setup for cloudfront. If that pans out, would be good to use the https links instead.

**Comment by GoodBoyDigital** [comment]
Nice! I will merge this for for Wednesdays release and open an issue for us to update the urls to HTTPS when its ready. Cheers!

File: README.md [hunk 2]
```diff
@@ -43,9 +43,20 @@ Pixi.js has full [WebGL](https://en.wikipedia.org/wiki/WebGL) support and seamle
 - Chat: You can join us on [Gitter](https://gitter.im/pixijs/pixi.js) To chat about Pixi. We also now have a Slack channel. If you would like to join it please Send me an email (mat@goodboydigital.com) and I will invite you in.
 
 
-### Installation ###
+### Setup ###
 
-Pixi.js can be installed with [Bower](https://bower.io/#getting-started), [NPM](https://docs.npmjs.com/getting-started/what-is-npm) or simply using a content delivery network (CDN) URL to embed Pixi.js directly on your HTML page. 
+Its easy to get started with Pixi.js! Simple grab the pre-built versions from here:
```
Comment (englercj) [inline README.md]: `Its` -> `It's`
`Simple` -> `Simply`

File: README.md [hunk 3]
```diff
@@ -43,9 +43,20 @@ Pixi.js has full [WebGL](https://en.wikipedia.org/wiki/WebGL) support and seamle
...
+Release Branch - Nice and stable Pixi.js
+- Unminified: [http://pixijs.download/release/pixi.js]
+- Minified: [http://pixijs.download/release/pixi.min.js] 
+
+Develop Branch - The bleeding edge version of Pixi.js
+- Unminified: [http://pixijs.download/dev/pixi.js]
+- Minified: [http://pixijs.download/dev/pixi.min.js]
+
+
+Alternitvly Pixi.js can be installed with [Bower](https://bower.io/#getting-started), [NPM](https://docs.npmjs.com/getting-started/what-is-npm) or simply using a content delivery network (CDN) URL to embed Pixi.js directly on your HTML page.
```
Comment (englercj) [inline README.md]: `Alternitvly` -> `Alternatively,`
`NPM` -> `npm`

**Comment by englercj** [comment]
Just noticed this was merged, so added the changes manually :)

**Pull request merged**